app/scripts/modules/core/src/pipeline/config/PipelineRegistry.ts

```diff
@@ -242,16 +240,9 @@ export class PipelineConfigProvider {
     return this.getManualExecutionComponent(this.getTriggerConfig(triggerType)) !== null;
   }
 
-  // TODO: How do i make sure this works?
```

Comment:
`i` should be capitalized, i.e. `I`

Reply:
That's the reason I deleted it.

Reply:
What if you added it back?

Reply:
Then thankfully I have this comment to help remind me.